docs_app/content/guide/testing/marble-testing.md

```diff
@@ -139,36 +137,32 @@ const expected = '-- 9ms a 9ms b 9ms (c|)';
 // or
 // const expected = '-----------a 9ms b 9ms (c|)';
 
-const result = cold(input).pipe(
-  concatMap(d => of(d).pipe(
-    delay(10)
-  ))
-);
+const result = cold(input).pipe(concatMap((d) => of(d).pipe(delay(10))));
```

Comment:
Prettier makes this example more difficult to read. IMO, we should leave the formatting as-is. If you put `<!-- prettier-ignore -->` on the line before the snippet, Prettier should ignore it.